### DISCUSSION TITLE
How do I run vcpkg in debug mode via cmake integration?

### DISCUSSION THREAD

**Discussion by cpp977** [microsoft/vcpkg#26030] [Q&A] [answered]
When vcpkg is run from the command line it accepts a `--debug` flag.
How can I achieve the same when vcpkg runs in cmake integration?

Thanks for help!

**Comment by autoantwort** [comment, marked as answer]
Set `VCPKG_INSTALL_OPTIONS` to `--debug` https://github.com/microsoft/vcpkg/blob/master/docs/users/buildsystems/cmake-integration.md#vcpkg_install_options
